### PR TITLE
EL-1137: Start page updates

### PR DESCRIPTION
**Pull request by HettieS**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1137)

## What changed and why

Add new content to the start page. Also, add a 'related content' sidebar with the public legal aid calculator.

Add link tracking for the public legal aid calculator.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: app/services/external_link_service.rb [hunk 1]
```diff
@@ -75,12 +75,12 @@ def mapping
           legislation_ammendments: {
             page_url: "https://www.legislation.gov.uk/all?title=Civil%20Legal%20Aid%20%28Financial%20Resources%20",
           },
-          controlled_work_applications: {
```
Comment (HettieS) [inline app/services/external_link_service.rb:78]: have removed this link as it is a duplicate

File: config/locales/en.yml [hunk 2]
```diff
@@ -517,6 +517,8 @@ en:
         text: Guide to determining controlled work
       certificated_guidance:
         text: Guide to determining certificated work
+      legal_aid_checker_for_public:
```
Comment (patrick-laa) [inline config/locales/en.yml:520]: I think this is in the wrong place - as far as I can see it's only used on the start page, and this is the section for content for the level of help page.

**Review by patrick-laa — APPROVED** [review]
One small point about i18n placement, otherwise all looks good to me.

File: config/locales/en.yml (hunk 2 above)
Comment (HettieS) [reply config/locales/en.yml:520]: Woops! Yes totally, originally I had put the link in the wrong sidebar location, when I corrected it I forgot to move this - will fix

**Pull request merged**